pkg/trafficpolicy/trafficpolicy.go

```diff
@@ -89,13 +89,24 @@ func (out *OutboundTrafficPolicy) AddRoute(httpRouteMatch HTTPRouteMatch, weight
 }
 
 // MergeInboundPolicies merges latest InboundTrafficPolicies into a slice of InboundTrafficPolicies that already exists (original)
-func MergeInboundPolicies(original []*InboundTrafficPolicy, latest ...*InboundTrafficPolicy) []*InboundTrafficPolicy {
+func MergeInboundPolicies(isMergeWithIngress bool, original []*InboundTrafficPolicy, latest ...*InboundTrafficPolicy) []*InboundTrafficPolicy {
```

Comment:
It'd be nice to move toward decoupling packages as much as possible. Referencing ingress in this variable name makes the trafficpolicy package more coupled with other packages. Let's make this variable name more generic like `partialHostnameMatchOK` or something like that.

Reply:
Thanks for the suggestion, I have renamed the variable